### DISCUSSION TITLE
Expo up to date example

### DISCUSSION THREAD

**Discussion by ScreamZ** [microsoft/rnx-kit#2287] [Q&A] [answered]
Hello,

I'm working on a turbo-repo pnpm mono repository, and I would like to support symlinks for my app. This app is using Expo for react native, but I can't find example on how to make it work ?

Maybe anyone could help me ?

Thanks

**Comment by tido64** [comment, marked as answer]
Hi, Expo has some documentation on how to customize the Metro bundler: https://docs.expo.io/guides/customizing-metro

I haven't tested this, but you can probably do something like below:

```js
const { getDefaultConfig } = require('expo/metro-config');
const MetroSymlinksResolver = require('@rnx-kit/metro-resolver-symlinks');

const config = getDefaultConfig(__dirname);
config.resolver.resolveRequest = MetroSymlinksResolver();

module.exports = config;
```
